internal/jobs/update-recommendations.go

```diff
@@ -0,0 +1,145 @@
+package jobs
+
+/*
+This job looks for missing recommendations, requests and saves them
+Originally intended for integration with Kruize v20
+*/
+
+import (
+	"fmt"
+	"encoding/json"
+	"net/http"
+	"time"
+	"reflect"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+
+	"github.com/redhatinsights/ros-ocp-backend/internal/config"
+	"github.com/redhatinsights/ros-ocp-backend/internal/logging"
+	"github.com/redhatinsights/ros-ocp-backend/internal/utils"
+	"github.com/redhatinsights/ros-ocp-backend/internal/model"
+	"github.com/redhatinsights/ros-ocp-backend/internal/utils/kruize"
+	
+	database "github.com/redhatinsights/ros-ocp-backend/internal/db"
+)
+
+var log = logging.GetLogger()
+var cfg *config.Config = config.GetConfig()
+var db = database.GetDB()
+
+var (
+	invalidRecommendationUpdateRecommendationJob = promauto.NewCounter(prometheus.CounterOpts{
+		Name: "rosocp_invalid_recommendation_update_recommendation_job",
+		Help: "Invalid recommendations sum from Kruize gathered while executing update-recommendations job",
+	})
+)
+
+func checkURLStatus() bool {
+
+	url := cfg.KruizeUrl + "/updateRecommendations"
+    resp, err := http.Get(url)
```

Comment:
Have you tested this? I think kruize does not support `GET` method for` /updateRecommendations` API. So you will never get `200` response in return.

Reply:
I didn't want to repeat my testing cycles, additionally most of the code here is re-used. Hence the _Do Not Merge_ label.
Now that the new Kruize image is available, I'll get these minor aspects checked.

Reply:
Change pushed